lib/Drupal/drupal_helpers/Module.php

```diff
@@ -79,13 +95,19 @@ public static function disable($module, $disable_dependents = TRUE) {
    * Uninstalls a module.
    *
    * @param string $module
-   * @param bool $disable_dependents
+   *   Module name to uninstall.
+   * @param bool $disable_dependencies
+   *   Flag to disable module's dependencies. Defaults to TRUE.
    *
    * @return bool
-   *  - TRUE: Module was uninstalled successfully.
+   *   Returns TRUE if module was uninstalled successfully,
+   *   \DrupalUpdateException is thrown otherwise.
+   *
+   * @throws \DrupalUpdateException
+   *   Throws exception if module was not uninstalled.
    */
-  public static function uninstall($module, $disable_dependents = TRUE) {
-    self::disable($module, $disable_dependents);
+  public static function uninstall($module, $disable_dependencies = TRUE) {
```

Comment:
Please revert the change to this function signature. Replacing "dependents" to "dependencies" changes the meaning.

See `module_enable` and `module_disable` in API docs - https://api.drupal.org/api/drupal/includes!module.inc/function/module_disable/7

```php
function module_disable($module_list, $disable_dependents = TRUE) {
  ..
}
```